acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
Yes, this is broken. These repos do exist, but they have a different format. My comment above, it turns out that value isn't used anywhere, which is why it doesn't break anything. This, on the other hand, is used, which is why it does break things. 

I'm hesitating because this method is removed in the latest release of beaker. What would it take for you to update to use the latest beaker version for testing?